pkgs/intl4x/lib/src/collation/collation.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../test_checker.dart';
+import 'collation_impl.dart';
+import 'collation_options.dart';
+
+class Collation {
+  final CollationOptions _options;
+  final CollationImpl _collationImpl;
+
+  const Collation(this._options, this._collationImpl);
+
+  /// Compare two strings in a locale-dependant manner.
+  ///
+  /// The [usage] can specify whether to use this for searching for a string,
```

Comment:
Here and in a few other references in this comment, we should reference the field in the options class.

```suggestion
  /// The [CollationOptions.usage] can specify whether to use this for searching for a string,
```